orca-clouddriver/src/main/java/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/DetermineRollbackCandidatesTask.java

```diff
@@ -527,6 +530,7 @@ private static class StageData {
     public String credentials;
     public String cloudProvider;
     public String serverGroup;
+    public Integer timeout;
```

Comment:
no need to define timeout here if you override `getDynamicTimeout`

Reply:
addressed